.github/workflows/mock_acpi.yml

```diff
@@ -34,14 +34,16 @@ jobs:
           ref: ${{ steps.pr_branch.outputs.head_sha }}
 
       - name: metal-runner-action
-        uses: equinix-labs/metal-runner-action@v0.3.0
+        uses: vprashar2929/metal-runner-action@custom-action
+        # uses: equinix-labs/metal-runner-action@v0.3.0
```

Comment:
it would be good if we have the custom `metal-runner-action@custom-action` in sustainable-computing org